src/main/java/com/faforever/client/game/CustomGamesController.java

```diff
@@ -96,14 +110,22 @@ public TilesSortingOrder fromString(String string) {
     ObservableList<Game> games = gameService.getGames();
 
     filteredItems = new FilteredList<>(games);
-    filteredItems.setPredicate(OPEN_CUSTOM_GAMES_PREDICATE);
+    showModdedGamesCheckBox.selectedProperty().bindBidirectional(preferencesService.getPreferences().showModdedGamesProperty());
+    showPasswordProtectedGamesCheckBox.selectedProperty().bindBidirectional(preferencesService.getPreferences().showPasswordProtectedGamesProperty());
+
+    updateFilteredItems();
+    preferencesService.getPreferences().showModdedGamesProperty().addListener(new WeakChangeListener<>(filterConditionsChangedListener));
+    preferencesService.getPreferences().showPasswordProtectedGamesProperty().addListener(new WeakChangeListener<>(filterConditionsChangedListener));
+
+    preferencesService.getPreferences().showModdedGamesProperty().addListener(new WeakChangeListener<>(showModsColumnChangedListener));
```

Comment:
What about the password column?

Reply:
Its small about we could also make that one disappear no? Did you test if that looks ok if they disappear and come back or if that messes up the column widths?

Reply:
yea, I like it better with password column still there as it gives some margins to the border of the table. But latest commit removes pw column as well,. So if you want to check out how it looks, please do